test-it/ExpectedResults/TestVI_VariantEquality_AttributeComparison.vtr

```diff
@@ -1,3 +1,6 @@
+TestResult_EmptyVariantConstantWithEqualDataAndUnequalAttributeNameAndValue true
```

Comment:
![emptyvariants](https://user-images.githubusercontent.com/2609749/56841611-73bfc500-6854-11e9-93eb-4186e78f076c.PNG)

Reply:
I also added a test that compares empty variant constant vs variant created using ToVariant.

Reply:
Do we have tests that cover empty variant constant vs variant with data?